### ISSUE TITLE
If no optional params are passed, 'No valid properties' error is thrown.

### ISSUE THREAD

**Issue by awolden** [dugword/valid-props#5]
If you define an object schema that is nothing but optional parameters it should allow an empty object to be successfully validated, instead it throws an error and essentially requires that the validated object ALWAYS have at least one parameter.

```javascript
let params;
  try {
    params = validator.validate({ /* Empty obj */ }, {
      option1: 'string?',
      option2: 'string?',
    });
  } catch (error) {
   // 'No valid properties' error is thrown here
  }

**Comment by dugword** [comment]
Thank you kind sir for your excellent feedback. Our developers will begin looking into this issue right away.

**Comment by dugword** [comment]
Resolved in 2.0.2
